### DISCUSSION TITLE
CSS compiled incorrectly for production in NextJS

### DISCUSSION THREAD

**Discussion by Lite5h4dow** [tailwindlabs/tailwindcss#3783] [Help] [answered]
I'm getting different results with Next JS dev and production after following the tutorial on the tailwind site.

Im compiling out very different CSS files for the development server and the static export of Next JS.

Gist here: https://gist.github.com/Lite5h4dow/0f7588d45f91ba093c73e863c8087f61

the static is missing the used definitions in the react page file, and i dont know how to combat this.

i followed the instructions to the letter and the files are showing up, its just missing stuff. i even tried the new template stuff from the next js template to no avail

**Comment by simonswiss** [comment, marked as answer]
Hey!

Are you able to share your Tailwind config file? This is probably where you have something set up incorrectly - likely in your `purge` settings.

Ideally, can you reproduce your setup in a minimal GitHub repo? That's the easiest way to see what is happening and provide help 👍

**Reply by Lite5h4dow** [reply]
Here is my tailwind.config.js
I attempted to replicate this in the Next JS template and no luck, however i copy and pasted those configs, so it should be working no?
```js
module.exports = {
  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {
    extend: {},
  },
  plugins: [],
};
```

**Reply by Lite5h4dow** [reply]
completely correct, knowing where to look i didnt add `./src/pages `/ `./src/components` my bad! ty!
